docs/BR.md

```diff
@@ -853,30 +853,26 @@ The CA SHALL update and reissue CRLs at least (i) once every twelve months and (
 No stipulation.
 
 ### 4.9.9 On-line revocation/status checking availability
-OCSP responses MUST conform to RFC6960 and/or RFC5019. OCSP responses MUST either:
-
-1. Be signed by the CA that issued the Certificates whose revocation status is being checked, or
-2. Be signed by an OCSP Responder whose Certificate is signed by the CA that issued the Certificate whose
-revocation status is being checked.
-
-In the latter case, the OCSP signing Certificate MUST contain an extension of type id-pkix-ocsp-nocheck, as
-defined by RFC6960.
+The CA SHALL provide an OCSP responder that conforms to the profile specified by RFC5019 for all Certificates issued, including the transport profile.
 
 ### 4.9.10 On-line revocation checking requirements
-Effective 1 January 2013, the CA SHALL support an OCSP capability using the GET method for Certificates issued
-in accordance with these Requirements.
+All OCSP responses MUST conform to the profile specified in Section 7.3
 
-For the status of Subscriber Certificates:
+For newly issued Subscriber Certificates, OCSP responses SHOULD be published prior to the delivery of the Certificate to the Subscriber. OCSP responses MUST be published no later than 10 minutes after the Subscriber Certificate has been issued.
 
-The CA SHALL update information provided via an Online Certificate Status Protocol at least every four days. OCSP responses from this service MUST have a maximum expiration time of ten days.
+The CA SHALL publish an updated OCSP response for Subscriber Certificates by the earlier of:
+1. 8 hours prior to the nextUpdate field of the previously published OCSP response
+2. If the OCSP response has a validity period, as defined by the difference in time between the thisUpdate and nextUpdate fields, of greater than 16 hours, then the value of the thisUpdate field plus one half of the validity period of the OCSP response.
 
-For the status of Subordinate CA Certificates:
+The CA SHALL publish an updated OCSP response for Subordinate CA Certificates by the earlier of:
+1. 24 hours after revoking a Subordinate CA Certificate.
+2. 365 days
 
-The CA SHALL update information provided via an Online Certificate Status Protocol at least (i) every twelve months and (ii) within 24 hours after revoking a Subordinate CA Certificate.
+If a Delegated OCSP Responder Certificate is used, the Responder Certificate MUST be directly signed and issued by the Issuer Certificate of the Subscriber Certificate it provides status information for. The Delegated OCSP Responder Certificate MUST contain the id-pkix-ocsp-nocheck extension as defined in RFC6960. The Delegated OCSP Responder Certificate MUST NOT have a Validity Period greater than 365 days, and SHOULD NOT have a validity period greater than 90 days.
```

Comment:
This starts off by repeating the language and intent of RFC 6960 [Section 2.6](https://tools.ietf.org/html/rfc6960#section-2.6) . The requirement for id-pkix-ocsp-nocheck is simply transplanted from the original line 862-863.

New to this is the introduction of a maximum bound of 365 days, which is otherwise unspecified, because a Responder Certificate is not-a Subscriber Certificate and (done properly), not-a Subordinate CA certificate.

Arguably, this profile better belongs in a section describing the types of certificates (Root CA, Subordinate CA, Delegated OCSP Responder, Test Certificate, Subscriber Certificate), but that's for a different ballot :)

The MUST matches CRLs 'worst case' validity period, which is 12 months. The SHOULD represents an aggressive, but hopefully not too aggressive, balance of the security risks that come from this.